convention-develocity-gradle-plugin/plugins/gradle-2-through-4/settings.gradle

```diff
@@ -1,7 +1,29 @@
 plugins {
```

Comment:
I don't understand the motivation of this PR. This is an example of how to build a convention plugin. I don't see the point in adding the DV configuration to it so that the project itself publishes build scans.

Reply:
Resolving this comment given the approach has changed.